Feature canvas scroll positioning

mikenikles opened the pull request:
Adds the ability to set a default scroll position when an image is zoomed in.

Status: **Ready for review**

Reviewers: @marlowpayne @jnurse @donnielrt 
## Changes
- Added a new `canvasScrollPosition` options object
- Specified default values for the above
- Updated README.md
- Bumped minor version
## Notes
- The default behaviour is kept by setting both scroll directions to _center_

Commit:
Add canvas scroll positioning

Commit:
Bump minor version

marlowpayne commented:
Currently in Magnifik, when a user taps on the thumbnail image, the canvas will open centered relative to where the user tapped on the thumbnail (user taps top left, canvas opens zoomed in to top left). 

Does this PR change that behaviour? If it does, is there a way we can keep both use cases?

Commit:
Replace the center option with a default option to reflect Magnifik's default behaviour

mikenikles commented:
Thanks @marlowpayne! I've updated the code to reflect that and also made sure that option is enabled by default to keep backwards compatibility.

I also added @donnielrt to get his feedback.

donnielrt commented on src/magnifik.js:
```diff
@@ -211,10 +216,25 @@ Mobify.UI.Magnifik = (function() {
-        thus.$canvas.prop('scrollTop', Math.max(0, Math.min(bigHeight - smallHeight,
-            bigHeight * topRatio - smallHeight / 2)));
+        // Set the desired "scrollLeft" position
+        if (this.options.canvasScrollPosition.horizontal === 'left') {
```
we might want to cache these properties to `var horizontal = this.options.canvasScrollPosition.horizontal`, and `var vertical = this.options.canvasScrollPosition.vertical`

donnielrt commented:
@mikenikles was this intended to solve a project-specific issue? Curious what the motivation for the change was?

mikenikles commented:
Yes @donnielrt, this PR is a generalized approach to solve a project-specific issue that may be encountered by other users of Magnifik.

donnielrt commented:
Awesome, thanks @mikenikles! Like we discussed, let's get a quick round of QA for the change in the project, and we can get this bad boy merged!
